kernel/kernel-defaults/src/test/scala/io/delta/kernel/defaults/DeltaReplaceTableSuite.scala

```diff
@@ -668,7 +671,11 @@ class DeltaReplaceTableSuite extends DeltaReplaceTableSuiteBase {
     }
   }
 
-  test("Column mapping maxFieldId is preserved during REPLACE TABLE") {
+  test("Column mapping maxFieldId is preserved during REPLACE TABLE " +
```

Comment:
This was actually a bug before -- we were accidentally not blocking going from (id -> none) when there were no property updates